### PR TITLE
Decode GIFs as static images on Android

### PR DESCRIPTION
**Pull request by redstar504**
### Problem

https://github.com/Expensify/App/issues/14751

Platform: Android
GIFs are not being decoded as static images, and therefore are resulting in a decoding error when uploading avatars.

[Screencast from 2023-02-13 04:28:30 AM.webm](https://user-images.githubusercontent.com/1311325/218460333-9d5d2605-cd54-4026-8ddc-e154ecd251f4.webm)

### Solution

Use the `ForceStaticImage` option from the Fresco decoder to decode the first frame only in a GIF image.

### Result

[Screencast from 2023-02-13 04:34:10 AM.webm](https://user-images.githubusercontent.com/1311325/218460677-909c9f29-d385-4b7d-9c09-ce8a77e2f153.webm)

### CODE REVIEW

**Review by mananjadhav — APPROVED** [review]

**Comment by mananjadhav** [comment]
@Beamanator I don't think we have a template for forked PRs right?

I've approved the PR and take care of the checklist in the App PR.

**Review by mananjadhav — COMMENTED** [review]
@redstar504 I think we would have to update the package.json version too. @Beamanator should we update this as a bugfix?

**Comment by redstar504** [comment]
@mananjadhav That is correct. Looks like the package spec in the app is based on the merge commit hash, so we will have to wait until this is merged to open the other other PR.

**Comment by mananjadhav** [comment]
@redstar504 I meant the package.json for `react-native-image-manipulator`. To further clarify, we're fixing a bug or adding a change, so we would publish a new version for the forked lib on npm. We'll then update this new version of the package dependency in the App package.json. I hope it makes sense :)

**Comment by redstar504** [comment]
@mananjadhav Yes it makes sense, but the current package.json is only referencing the commit of the last fix that was merged into this fork, so I am not sure we need to worry about versioning:
`"@oguzhnatly/react-native-image-manipulator": "github:Expensify/react-native-image-manipulator#c5f654fc9d0ad7cc5b89d50b34ecf8b0e3f4d050",`

We can just update the hash once this change is merged.

**Comment by mananjadhav** [comment]
Yeah I saw, I would then wait for @Beamanator to comment here. We do that for some of the other dependencies, I am not sure about this one. Can you please add @Beamanator as a reviewer please?

**Comment by redstar504** [comment]
@Beamanator we're ready for review.

**Comment by Beamanator** [comment]
It's a bit weird I wasn't automatically added as a reviewer, I'll have to look into why 👍 

@mananjadhav true we don't have PR templates for forks I believe, I wonder if this is something we should look into! Maybe ask in #contributor-plus if you feel like it, otherwise I'll try to remember to bring it up later this week when I'm back full time :D

I currently see what @redstar504 is saying and I believe we should continue just using the latest commit hash in App's `package.json` - though it's a bit funky that some forks we publish, some we don't. @mananjadhav do you mind starting a convo in #expensify-open-source about this? If you're busy no worries I can start the conversation but it will probably wait till Wednesday

**Review by Beamanator — APPROVED** [review]

**Comment by Beamanator** [comment]
For now @redstar504 can you please proceed with an App PR referencing the latest commit from this fork in `package.json`? 🙏

**Pull request merged**
